cobalt/black_box_tests/tests/preload_font.py

```diff
@@ -34,14 +34,16 @@ def test_preload_font(self):
       url = server.GetURL(file_name='testdata/preload_font.html')
 
       with self.CreateCobaltRunner(
-          url=url, target_params=['--preload']) as runner:
+          url=url,
+          target_params=['--preload'],
+          web_server_port=server.GetPort()) as runner:
         runner.WaitForJSTestsSetup()
         runner.SendResume()
         start_time = time.time()
         while runner.IsInPreload():
           if time.time() - start_time > _MAX_RESUME_WAIT_SECONDS:
-            raise Exception('Cobalt can not exit preload mode after receiving'
-                            'resume signal')
+            raise ValueError('Cobalt can not exit preload mode after receiving'
+                             'resume signal')
```

Comment:
Missing space between `receiving` and `resume`.

Not your fault but you might as well fix it here.